packages/cloud_firestore/cloud_firestore/example/test_driver/load_bundle_e2e.dart

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:typed_data';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:cloud_firestore_platform_interface/cloud_firestore_platform_interface.dart';
+import 'package:http/http.dart' as http;
+
+void runLoadBundleTests() {
+  group('$DocumentReference', () {
+    late FirebaseFirestore firestore;
+    late Uri url;
+    late Uint8List buffer;
+    const String collection = 'firestore-bundle-tests';
+    setUpAll(() async {
```

Comment:
It probably should be `setUp` instead of `setupAll`, and use a different FirebaseApp between each test (will need a unique name)

Reply:
We can keep a `setupAll` for the `buffer`. But it'd be great to wrap it in an `UnmodifiableListView` to make sure tests don't do anything fancy

Reply:
In fact it probably makes more sense to move the bundle logic inside the `loadBundle()` test